_posts/2021-08-18-cockpit-251.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Cockpit 251
+author: garrett
+date: '2021-08-18'
+tags: cockpit, machines
+slug: cockpit-251
+category: release
+summary: ''
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 251 and cockpit-machines 250:
+
+
+## Certificate/smart card authentication to ssh and sudo
+
+Once you logged into Cockpit with a certificate, you likely need to switch to administative mode, or connect to remote machines through SSH. If your machine is part of a FreeIPA domain, you can now set up [constrained delegation rules](https://www.freeipa.org/page/V4/Service_Constraint_Delegation) so that the Cockpit certificate authentication gets forwarded to sudo and/or ssh. This is particularly useful if your user account does not have a password. Please see the [Client certification documentation](https://cockpit-project.org/guide/latest/cert-authentication.html) for details.
```

Comment:
```suggestion
Once you've logged into Cockpit with a client certificate, you'll likely need to switch to administative mode, or connect to remote machines through SSH. If your machine is part of a FreeIPA domain, you can now set up [constrained delegation rules](https://www.freeipa.org/page/V4/Service_Constraint_Delegation) so that the Cockpit certificate authentication gets forwarded to sudo and/or ssh. This is particularly useful if your user account does not have a password. Please see the [Client certification documentation](https://cockpit-project.org/guide/latest/cert-authentication.html) for details.
```